### PR TITLE
Feature/nations detail screen 0.3.2

### PR DESCRIPTION
**Pull request by seland**
#### Changes
Nation details screen UI is updated.
Actions on drafts are implemented. 
Editing on drafts is implemented.

### CODE REVIEW

File: src/screens/NationDetailsContainer/index.js [hunk 1]
```diff
@@ -36,25 +107,31 @@ class NationDetailsContainer extends Component {
     if (_.isEmpty(this.props.wallets)) {
       this._showCreatePrivateKeyAlert();
     } else {
-      fn()
+      fn();
```
Comment (florianlenz) [inline src/screens/NationDetailsContainer/index.js]: Can we rename `fn`?

File: src/screens/NationDetailsContainer/index.js [hunk 2]
```diff
@@ -36,25 +107,31 @@ class NationDetailsContainer extends Component {
     if (_.isEmpty(this.props.wallets)) {
       this._showCreatePrivateKeyAlert();
     } else {
```
Comment (florianlenz) [inline src/screens/NationDetailsContainer/index.js]: This else is redundant as far as I can see. 
You can have just: 
```
if (_.isEmpty(this.props.wallets)) {
   this._showCreatePrivateKeyAlert();
}

fn()
```

Comment (seland) [reply src/screens/NationDetailsContainer/index.js]: We should call function only if user has wallet, so we need either else or return inside if

File: src/screens/NationDetailsContainer/index.js (hunk 1 above)
Comment (seland) [reply src/screens/NationDetailsContainer/index.js]: Done.

File: src/screens/NationDetailsContainer/index.js (hunk 2 above)
Comment (florianlenz) [reply src/screens/NationDetailsContainer/index.js]: Ah yeah of course. But go for a return. It's less code.

**Pull request merged**